src/software/world/robot.h

```diff
@@ -116,14 +116,14 @@ class Robot
      *
      * @return the missing capabilities of the robot
      */
-    const std::set<RobotCapability> &getCapabilitiesBlacklist() const;
+    const std::set<RobotCapability> &getUnavailableCapabilities() const;
 
     /**
      * Returns all capabilities this robot has
      *
      * @return Returns all capabilities this robot has
```

Comment:
IMO, javadoc should be updated to "Returns available capabilities", since `allRobotCapabilities` in `robot_capabilities.h` already returns "all capabilities", so this would be more and accurate.

Reply:
Fixed!